README.md

```diff
@@ -17,26 +17,26 @@ Flashbots Research and Product are tightly-coupled dual engines that propel our
 
 Our work is open source and you can follow our progress in each respective repository of the Flashbots organization. Anyone is welcome to comment on an issue or pick it up entirely.
 
-### Community
-The best way to interact with the Flashbots community is through our [forum](https://collective.flashbots.net/), which is our central communication platform for conversations, announcements and information sharing. We also publish monthly [transparency reports](https://collective.flashbots.net/search?q=transparancy%20report%20%23ship%20in%3Atitle) that aggregates information and provides summaries on everything Flashbots and MEV.
+## Community
+The best way to interact with the Flashbots community is through our [forum](https://collective.flashbots.net/), which is our central communication platform for conversations, announcements and information sharing. Additionally we hold recurring [community calls](./commnuity-calls/README.md) on a variety of topics!
 
-### Join us!
+## Join us!
 We are not your typical project, we are fully remote and our principles are based on that of a [pirate hacker collective](https://www.youtube.com/watch?v=T0fAznO1wA8). If you are a self-directed individual who puts collective success above your own and is motivated by solving challenging problems with asymmetric impact, you will fit right in. 
-* [**Open job positions**](https://boards.greenhouse.io/flashbots): Check out the list of full-time roles we're actively recruiting for.
+* [**Open job positions**](https://www.flashbots.net/jobs): Check out the list of full-time roles we're actively recruiting for.
 * [**MEV research fellowship**](https://github.com/flashbots/mev-research/blob/main/grants.md):  We issue research grants to Flashbots Research Proposal submitters. 
 * For anything else, please reach out to us at [jobs@flashbots.net](mailto:jobs@flashbots.net)!
 
-**Useful links**
+## Useful links
 * 🌐 Website: https://flashbots.net
 * 📄 Docs: https://docs.flashbots.net
+* ☀️ SUAVE Docs: https://suave-alpha.flashbots.net/
 * 📡 Status: https://status.flashbots.net
 * 💬 Discord: https://discord.gg/flashbots
 * 💡 Forum: https://collective.flashbots.net
 * 📝 Writings: https://writings.flashbots.net
 * ⚡ MEV-Boost: https://boost.flashbots.net
-* 🗓️ Calendar: https://calendar.flashbots.net
```

Comment:
Why remove the calendar? Seems useful to me

Reply:
oh whoops didnt try this